README.md

```diff
@@ -20,6 +20,20 @@ The format of the reports is currently either text or HTML.
 
 ## Usage
 
+There is a Dockerfile and a Makefile supplied in this repo. All you need to do is:
+
```

Comment:
I think I would prefer this in a separate section to the usage as its not required to use Docker to build or run the tool.

Reply:
my point with this PR is to make it easy for people to use the tool right away; especially for folks who have zero java experience, such my myself -- I literally had to google how to compile java sources; I can write about that and put it in a different section, that's not a problem